.deployment.config.json

```diff
@@ -16,55 +16,55 @@
   },
   "ordered_phases": [
     {
-      "id": "deploy-headless-to-s3-latest",
+      "id": "deploy-headless-patch-to-s3-version",
```

Comment:
I re-ordered the phases in this step so that it's : 
* headless patch -> minor -> major -> latest
* atomic patch -> minor -> major -> latest
* atomic/headless patch -> minor -> latest